config/apply/ccsm/translate.js

```diff
@@ -282,6 +284,26 @@ const customEccCodes = {
   }
 };
 
+// Extract Procedures from FindingType
+// ## FindingType
+// >Can be null.
+
+// Maps to Category values in ECT, 14001 - CCS - TRANSCRIBED FINDING TYPES
+
+// ECT.14001.1 - Pap Smear
+// ECT.14001.2 - HPV
+// ECT.14001.3 - Transformation Zone
+// ECT.14001.4 - Colposcopy
+// ECT.14001.5 - Excision
+// ECT.14001.6 - Ablation
+// ECT.14001.7 - Endometrial Biopsy
+// ECT.14001.8 - Endocervical Curettage
+const procedureMappings = {
+  'ECT.14001.4': 'Colposcopy',
+  'ECT.14001.5': 'Cervix Excision',
+  'ECT.14001.6': 'Cervical Histology'
```

Comment:
I believe this `ECT.14001.6` should map to `'Cervix Ablation'`, to match https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/main/fsh-tank/fsh-generated/resources/ValueSet-PertinentProcedureShortList.json#L59, which is the key to query the value set where we're pulling the procedure codes from on line 5. 

`/node_modules/ccsm-cds-with-tests/dist/fhir/ValueSet/PertinentProcedureShortList.js` is the version of that value set pulled in by npm. You'll see that there is a 'Cervix Ablation' display field that we want to access, but no 'Cervix Histology'.